index.html

```diff
@@ -240,5 +240,34 @@ <h3>Skills</h3>
         </li>
       </ul>
     </section>
+
+    <section class="form-container">
+         <div class="form-text">
+           <h3>Contact me</h3>
```

Comment:
You did not address the previous reviewer's requirement about the section's background image. Please do so.